src/main/java/de/dhbw/karlsruhe/students/mailflow/external/infrastructure/cli/MainCLIPrompt.java

```diff
@@ -78,12 +80,13 @@ private BaseCLIPrompt showActionMenuPrompt() {
   public void start() {
     super.start();
     while (true) {
-
       while (!collectionAuth.authSession().isLoggedIn()) {
         BaseCLIPrompt registerOrEmailPrompt = showRegisterOrEmailPrompt();
         registerOrEmailPrompt.start();
       }
 
+      WorkerQueue.getInstance().performDueJobs();
+
```

Comment:
is this needed here when you do super.start()?

Reply:
I think so.

It might be that it is required for a job to be processed on time, otherwise the action menu prompt could show outdated information.